info/container.go

```diff
@@ -467,6 +467,8 @@ type Usage struct {
 type DerivedStats struct {
 	// Time of generation of these stats.
 	Timestamp time.Time `json:"timestamp"`
+	// Latest instantaneous sample.
+	LatestUsage Usage `json:"latest_usage"`
```

Comment:
I was debating putting in a new type here with just one value. We can do that - one uint64 each for cpu and memory.

The goal was to extend the derived stats from day, hour, minute down to second. When cadvisor starts, you just receive a bunch of zeroes for a minute. With seconds data, caller can start seeing usage stat earlier.